.isort.cfg

```diff
@@ -2,7 +2,6 @@
 line_length=79
 indent='    '
 multi_line_output=3
-known_third_party=pytz,cffi,lark-parser
```

Comment:
Is there a reason for removing the `known_third_party` option? Is it optional with `isort5`?

Reply:
Yep, now by default it's a third party. Only first party are relevent now.